lms/djangoapps/courseware/url_helpers.py

```diff
@@ -43,4 +46,8 @@ def get_redirect_url(course_key, usage_key):
             'courseware_position',
             args=(unicode(course_key), chapter, section, navigation_index(position))
         )
+
+    if final_target_id:
+        redirect_url += "?activate_block_id={final_target_id}".format(final_target_id=final_target_id)
```

Comment:
This needs to be url encoded. Right now, the `jump_to_id` links aren't working with split modulestore because the `+` characters in the block ID are getting changed to spaces.

![screen shot 2015-07-03 at 12 42 41 pm](https://cloud.githubusercontent.com/assets/945577/8504155/05c4b0e2-2181-11e5-9e19-c96d8ee26da2.png)